Userland/Applications/Browser/BookmarksBarWidget.cpp

```diff
@@ -140,7 +140,8 @@ BookmarksBarWidget::BookmarksBarWidget(const String& bookmarks_file, bool enable
     }));
     m_context_menu->add_action(GUI::CommonActions::make_delete_action([this](auto&) {
         remove_bookmark(m_context_menu_url);
-    }));
+    },
+        nullptr, true));
```

Comment:
This alignment is :acid2: but that's how Clang-format likes it apparently!

Reply:
You can move the first lambda to a new line, the formatting will be much nicer with that.

Reply:
:+1: Thanks, it's a little better!